docs/operator-manual/server-commands/argocd-application-controller.md

```diff
@@ -42,8 +42,10 @@ argocd-application-controller [flags]
       --metrics-port int                       Start metrics server on given port (default 8082)
   -n, --namespace string                       If present, the namespace scope for this CLI request
       --operation-processors int               Number of application operation processors (default 10)
-      --otlp-address string                    OpenTelemetry collector address to send traces to
-      --otlp-attrs strings                     List of OpenTelemetry collector extra attrs when send traces, each attribute is separated by a colon(e.g. key:value)
+      --otlp-address string                     OpenTelemetry collector address to send traces to
+      --otlp-attrs strings                      List of OpenTelemetry collector extra attrs when send traces, each attribute is separated by a colon(e.g. key:value)
+      --otlp-headers stringToString             List of OpenTelemetry collector extra headers sent with traces, headers are comma-separated key-value pairs(e.g. key1=value1,key2=value2) (default [])
+      --otlp-insecure                           OpenTelemetry collector insecure mode (default true)
```

Comment:
I think the issue with the failing CI is too much indentation

```suggestion
      --otlp-address string                   OpenTelemetry collector address to send traces to
      --otlp-attrs strings                    List of OpenTelemetry collector extra attrs when send traces, each attribute is separated by a colon(e.g. key:value)
      --otlp-headers stringToString           List of OpenTelemetry collector extra headers sent with traces, headers are comma-separated key-value pairs(e.g. key1=value1,key2=value2) (default [])
      --otlp-insecure                         OpenTelemetry collector insecure mode (default true)
```

In case that doesn't work, running `make clidocsgen` should hopefully do the trick 🙏

Reply:
@blakepettersson Thanks for pointing that out. Resolving it.

Reply:
Updated.